templates/blog-article/blog-article.js

```diff
@@ -15,7 +15,7 @@ export default async function decorate(main) {
     subHeading = main.querySelector('strong>p');
   }
   let headingSection = div(h1);
-  if (subHeading && subHeading.innerText === subHeading.closest('p').innerText) {
+  if (subHeading && subHeading.innerText === subHeading.closest('p').innerText && h1.nextSiblingElement === subHeading) {
```

Comment:
Can you put the subheading after the H1?
<img width="1092" alt="image" src="https://github.com/user-attachments/assets/ed64e236-4931-4b58-a12e-d21e498a3c2a">